content/applications/sales/sales/send_quotations/get_signature_to_validate.rst

```diff
@@ -1,51 +1,52 @@
-===================================
-Get a signature to confirm an order
-===================================
+=========================================
+Online signatures for order confirmations
+=========================================
 
-Online signatures are like electronic "fingerprints". By using them on Odoo, you will get
-automatic orders confirmation. You and your customers will save a lot of time by using this
-feature compared to a traditional process.
+The Odoo *Sales* application provides customers with the opportunity to confirm orders via an
+online signature directly on the sales order. Once electronically signed by the customer, the
+salesperson attached to the sales order is automatically notified that the order is confirmed.
 
-Activate online signature
-=========================
+Activate online signatures
+==========================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Signature**
-feature.
+To activate the *Online Signature* feature, go to :menuselection:`Sales app --> Configuration -->
+Settings`, scroll to the :guilabel:`Quotations & Orders` heading, and activate the
+:guilabel:`Online Signature` feature by checking the box beside it, and clicking the
+:guilabel:`Save` button in the top-left corner.
 
-.. image:: get_signature_to_validate/signature_1.png
+.. image:: get_signature_to_validate/signature-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable online signature on Odoo Sales?
+   :alt: How to enable online signature on Odoo Sales.
 
 .. note::
-   If you are familiar with this documentation: :doc:`quote_template`, you can activate or not the
-   **Online Signature** feature for each quotation template you use, under their confirmation tab.
-   Example:
+   If you are familiar with the documentation explaining :doc:`quote_template`, the
+   :guilabel:`Online Signature` feature for each quotation template is found under the
+   :guilabel:`Confirmation` tab.
```

Comment:
This conditional doesn't make sense. It reads like you're saying that the Online Signature feature is found on the Confirmation tab IF the reader is familiar with the quote template documentation. I think you can remove the intro entirely. See below.
```suggestion
   The :guilabel:`Online Signature` feature for each quotation template is found under the
   :guilabel:`Confirmation` tab.
```